File: RedSheet.DbEntities/Models/vProjectModuleReviewRecord.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vProjectModuleReviewRecords", Schema = "dbo")]
	[Serializable]
	public partial class vProjectModuleReviewRecord  
	{
 
	[Required]
	public string Feedback { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int ProjectModuleId { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Range(1, int.MaxValue)]
	public int ProjectModuleReviewId { get; set; } 

	public vProjectModuleReviewRecord()
	{
    }

    }
}

File: redsheet-portal/src/app/database-models/v-project-module-review-record.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vProjectModuleReviewRecord {
    constructor(vProjectModuleReviewRecord?: vProjectModuleReviewRecord )  {
        let properties = [ "feedback", "projectModuleId", "projectModuleReviewId",];
        for (let property of properties)
            if (vProjectModuleReviewRecord && vProjectModuleReviewRecord[property])
                this[property] = vProjectModuleReviewRecord[property];
    }
 
    @required()
	feedback : string =   undefined;
 
	projectModuleId : number =   0 ;
 
    @range(1,2147483647)
	projectModuleReviewId : number =   undefined;


}
